src/molecules/Footer.tsx

```diff
@@ -0,0 +1,273 @@
+  margin: 14px 0;
+  display: flex;
+  justify-self: center;
+
+  @media (min-width: ${Layout.breakpoints.md.minWidth}) {
+    justify-self: end;
+  }
+`
+
+const StyledContact = styled.div`
+  color: ${({ theme }) => theme.colors.blackLight};
+`
+
+const StyledP = styled.p`
+  margin: 0;
+`
+
+const StyledLink = styled.a`
+  color: ${({ theme }) => theme.colors.greyLight};
+`
+
+const StyledUl = styled.ul`
+  margin: 0;
+  padding: 0;
+  list-style: none;
+`
+
+const StyledList = styled.li`
+  margin-bottom: 10px;
+`
+
+const StyledNumber = styled.p`
+  margin-top: 0;
+`
+
+const StyledEmail = styled.span`
+  border-bottom: 2px solid #f9d4bd;
+`
+
+const StyledContainer = styled.div`
+  margin-bottom: 45px;
+
+  @media (max-width: ${Layout.breakpoints.sm.maxWidth}) {
+    margin-bottom: 20px;
+    display: flex;
+    justify-content: center;
+  }
+`
+
+const StyledInput = styled.input`
+  background-color: rgba(255, 255, 255, 0.01);
+  border-style: solid;
+  border-width: 0 0 2px;
+  border-color: #ddd;
+  padding: 15px 0;
+  text-align: center;
+  width: 100%;
+
+  @media (min-width: ${Layout.breakpoints.md.minWidth}) {
+    text-align: left;
+  }
+`
+
+const StyledBtnSubmit = styled.button`
+  background-color: rgba(0, 0, 0, 0);
+  color: #222;
+  padding: 15px 10px;
+  font-size: 14px;
+  font-weight: 600;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  border-style: solid;
+  border-width: 0 0 2px;
+  border-color: #ddd;
+  text-align: center;
+
+  :hover {
+    color: ${({ theme }) => theme.colors.orange};
+    border-color: ${({ theme }) => theme.colors.orange};
+  }
+
+  @media (min-width: ${Layout.breakpoints.md.minWidth}) {
+    min-width: 130px;
+  }
+`
+
+const StyledContainerNewsletter = styled.div`
+  @media (min-width: ${Layout.breakpoints.md.minWidth}) and (max-width: ${Layout
+      .breakpoints.md.maxWidth}) {
+    text-align: center;
+  }
+`
+
+const StyledLinkSocial = styled.a`
+  color: #888;
+  padding-right: 15px;
+`
+
+export const Footer: React.FC = () => {
+  return (
+    <Composition
+      template={templateMobile}
+      templateMd={templateMd}
+      templatelg={templateDesktop}
+      paddingHorizontal="20px"
+      paddingHorizontalMd="40px"
+      paddingTop="60px"
+      paddingToplg="100px"
+      paddingBottom="25px"
+      as={StyledFooter}
+      gap={2}
+      flex
+      justifyContentSmDown="center"
+    >
+      {Areas => (
+        <>
+          <Areas.Contacts>
+            <StyledContainer>
+              <Logo src="https://demo2.wpopal.com/auros/wp-content/uploads/2019/02/Logo.svg" />
+            </StyledContainer>
+            <Composition as={StyledContact} gap={2} templateCols="1fr">
+              <div>
+                <StyledP>
+                  2593 Timbercrest Road, Chisana, Alaska Badalas
+                </StyledP>
+                <StyledP>United State</StyledP>
+              </div>
+              <div>
+                <StyledNumber>(+91)0-000-1111</StyledNumber>
```

Comment:
For such cases when you need to cancel some default behavior of an element (like setting `margin-top:0` in this case), I usually do:

```jsx
<Box as="p" marginTop={0}>
```

I don't have to create a `StyledNumber` in that case at all.